include/deal.II/base/mg_level_object.h

```diff
@@ -75,11 +75,9 @@ class MGLevelObject : public Subscriptor
                 Args &&...args);
 
   /**
-   * Constructor. Same as above but without arguments to be forwarded to the
-   * constructor of the underlying object.
+   * Default constructor.
    */
-  MGLevelObject(const unsigned int minlevel = 0,
```

Comment:
Since no argument is equivalent to template `<>` of the previous ctor, the only usage of this function is for default construction.